### PR TITLE
Added examples for progress control

### PR DESCRIPTION
**Pull request by FabienLelaquais**
+ removed 'color' property from progress control
+ added CSS class names for value and title in the control
+ minor refman issues

### CODE REVIEW

File: taipy/gui/_renderers/factory.py [hunk 1]
```diff
@@ -444,6 +444,24 @@ class _Factory:
...
+            [
+                ("linear", PropertyType.boolean, False),
+                ("show_value", PropertyType.boolean, False),
+                ("title", PropertyType.dynamic_string),
+                ("title_anchor", PropertyType.string, "bottom"),
+                ("render", PropertyType.dynamic_boolean, True),
+                ("width", PropertyType.string_or_number),
+            ]
+        )
+        ._set_propagate(),
```
Comment (FredLL-Avaiga) [inline taipy/gui/_renderers/factory.py]: don't think `propagate` is needed here as there's no variable modification from the frontend

**Review by FredLL-Avaiga — DISMISSED** [review]

**Review by namnguyen20999 — DISMISSED** [review]
lgtm

File: taipy/gui/_renderers/factory.py (hunk 1 above)
Comment (FabienLelaquais) [reply taipy/gui/_renderers/factory.py]: Let's discuss this rapidly. I'm sure you're right.

Comment (FabienLelaquais) [reply taipy/gui/_renderers/factory.py]: Done.

**Review by FredLL-Avaiga — DISMISSED** [review]

**Review by FredLL-Avaiga — APPROVED** [review]

**Pull request merged**